addons/web/static/src/js/libs/popper.js

```diff
@@ -0,0 +1,5 @@
+odoo.define('/web/static/lib/bootstrap/js/popover.js', function () {
+    "use strict";
+    //The pop-up will be overflowed on the right and bottom.
+    Popper.Defaults.modifiers.preventOverflow.priority = ['right', 'left', 'bottom', 'top'];
+});
```

Comment:
Can you explain exactly what this does, I am not sure to understand?

Reply:
if the biography is too large, it will be overflowed to the right and bottom, on the other hand, if the priority is not overridden, the pop-up will be overflowed to the left and top.
default: https://github.com/odoo/odoo/blob/a991a579bc53a087d05460aa01724b4a4af4be1b/addons/web/static/lib/popper/popper.js#L2068

Reply:
Ok, thanks :+1: